appcenter-auth/.eslintrc

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please add `appcenter-auth` directory to https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/scripts/eslint.sh#L7 so that CI will run eslint on `appcenter-auth` directory.

We should do the same for `appcenter-data` module, but that can be done by a separate PR.

Reply:
I'll do a separate PRs to update the script files